### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by NikkiMeganMoore**
The purpose of this PR is to review the codebase as a whole. It does **not** need to be merged, but all of the comments should be addressed in master. 

A lot of commented out code was found and should be removed so it doesnt cause unnecessary clutter. Other things that will need to be removed are any console.logs that are not for the purpose of debugging unresolved issues on prod, and especially sensitive API key information and username/passwords that were found hardcoded. 

In addition to the above, we had some general comments and questions regarding various things such as what can be done at bundle time, code formatting, and certain code structure choices.

Please review all of the comments and apply changes to the master branch. Once the changes are all made we will want to work off of branches moving forward instead of directly off master and will be implementing a code review process for any changes made as per the requirements of SOC2. 

Thanks and let me know if you have any questions

### CODE REVIEW

File: test_page/src/App.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        this.setState({
+            isLogged: true,
+            authentication: {
+                ...values,
+                ...authentication
+            }
+        })
+        // const topics = getIntroMessageTopics(getActiveIntegrator(
+        //     this.datamessenger.options.authentication.domain
+        // ))
```
Comment (benWozak) [inline test_page/src/App.js:105]: remove/fix

File: test_page/src/DataMessengerPage/DataMessengerPage.js [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+        enableNotifications: true,
+        enableColumnVisibilityManager: true,
+        enableVoiceRecord: true,
+        dashboardTitleColor: 'rgb(72, 105, 142)',
+        clearOnClose: false,
+        height: 500,
+        width: 550,
+        title: 'Data Messenger',
+        lightAccentColor: '#26a7df',
+        // lightAccentColor: '#2466AE',
```
Comment (benWozak) [inline test_page/src/DataMessengerPage/DataMessengerPage.js:49]: remove

File: src/Charts/ChataChartSeriesPopover.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        tick.style.top = '3px';
+        tick.style.left = '23px';
+        checkboxInput.style.marginTop = '4.5px';
+        checkboxInput.style.marginLeft = '20px';
+
+        checkboxContainer.style.width = '38px';
+        checkboxContainer.style.height = '18px';
+        checkboxWrapper.style.width = '38px';
+        checkboxWrapper.style.height = '18px';
+        checkboxWrapper.style.position = 'relative';
```
Comment (NikkiMeganMoore) [inline src/Charts/ChataChartSeriesPopover.js:85]: Is there a reason this is done through JS instead of CSS?

File: src/Charts/ChataColumnChart.js [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+    var colStr2 = cols[index1]['display_name'] || cols[index1]['name'];
+
+    var col1 = formatColumnName(colStr1);
+    var col2 = formatColumnName(colStr2);
+    const barWidth = width / data.length;
+    const rotateLabels = barWidth < 135;
+    const interval = Math.ceil((data.length * 16) / width);
+    var xTickValues = [];
+    var allLengths = [];
+    if (barWidth < 16) {
```
Comment (NikkiMeganMoore) [inline src/Charts/ChataColumnChart.js:103]: I noticed that some of the code formatting is not consistent. Do you use any linting or auto-formatters?  I highly recommend using prettier or eslint. Personally I have mine format the file every time I save so I never have to think about it.

File: src/Constants/Constants.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    '--chata-dashboard-background-color': '#fff',
+    '--chata-dashboard-border-color': '#d3d3d352',
+    '--chata-dashboard-hover-color': '#ececec',
+    '--chata-dashboard-text-color-primary': '#5d5d5d',
+    '--chata-dashboard-text-color-placeholder': '#0000009c'
+}
+
+
+
+export const MONTH_NAMES = [
```
Comment (NikkiMeganMoore) [inline src/Constants/Constants.js:42]: We should change this to not be hardcoded in english. We should just use dayjs to format the month name for us

**Pull request closed without merging**